lib/italian_dishes.json

```diff
@@ -0,0 +1,3 @@
+{
+  "Carbonara": 7, "Bolognese": 6, "Canneloni": 8, "Calamari": 9, "Arancini": 6, "Margherita": 7, "Hawaiian": 99
+}
```

Comment:
I like this, nice easy way of updating the menu without messing with any other code! 👍 Good for the Open-Closed SOLID pronciple